src/popup/lib/utils.test.ts

```diff
@@ -0,0 +1,63 @@
+import { formatNumber } from './utils'
+
+describe('formatNumber', () => {
+  it('should display right format for integers', () => {
+    let res = formatNumber(5, 2)
+    expect(res).toEqual('5.00')
```

Comment:
We can do this for every case:
```suggestion
    expect(formatNumber(5, 2)).toEqual('5.00')
```